runelite-client/src/main/java/net/runelite/client/plugins/itemskeptondeath/FixedPriceItem.java

```diff
@@ -219,6 +219,9 @@
 	VERACS_FLAIL_100(ItemID.VERACS_FLAIL_100, 10000, ItemID.VERACS_FLAIL_0),
 
 	AVERNIC_DEFENDER(ItemID.AVERNIC_DEFENDER, 4040000),
+
+	ETERNAL_TELEPORT_CRYSTAL(ItemID.ETERNAL_TELEPORT_CRYSTAL, 1),
```

Comment:
Would like some guidance on how to determine the exact offset value.

Reply:
Value `500` is taken from osrs wiki: https://oldschool.runescape.wiki/w/Eternal_teleport_crystal